views/components/Graphs/AddNewChart.jsx

```diff
@@ -16,7 +16,7 @@ const AddNewChart = () => {
       <Fab
         component="span"
         focusRipple
-        onClick={() => window.location.assign(routes.newChart)}
+        onClick={() => onNewChart(routes.newChart)}
```

Comment:
we don't want to pass `routes.newChart` here because it's just as easy for the parent to know the new chart route.

```suggestion
        onClick={onAddNewChart}
```